### ISSUE TITLE
Activity

### ISSUE THREAD

**Issue by johan-lejdung** [facebookgo/inject#17]
Would it be safe to assume that this project is dead from FB point of view?

**Comment by kdeenanauth** [comment]
I think it works great

**Comment by johan-lejdung** [comment]
Yeah, functionally it does. But it has pending PR's and Issues since June/October last year, with no comments.

**Comment by kdeenanauth** [comment]
I see - I'm just a avid user of the library. I would like to see a response regarding the license files. I don't see much of the pull requests getting merged because the library is very minimal in nature and the features being added are superfluous IMO.

**Comment by emanuel-v-r** [comment]
Actually if that is the case @kdeenanauth, they should still comment the PR, I made one yesterday, but regarding the other ones I dont even know If it would be reviewed or not.

**Comment by fantasist** [comment]
Most libraries in facebookgo were built by Parse, which was fully shutdown in Jan 2017. I'm not sure whether any facebook employee (i.e. ex Parsers) still has spare time to manage this repo. I have updated 3 projects with most watches from patented license to MIT license. So feel free to create community forks and merge pull requests there.

**Comment by emanuel-v-r** [comment]
Wich references to the original projecto should have in the fork? Can also use the documentation?

**Comment by fantasist** [comment]
@imaramos You don't have to reference the original project.

**Comment by emanuel-v-r** [comment]
About the documentation, its ok tu use it?

**Comment by fantasist** [comment]
This project only has Godoc, which is basically comments in source code. Feel free to use it.

**Comment by emanuel-v-r** [comment]
Thanks for the info!
